ci/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set version = "2.0.1" %}
```

Comment:
I'm unusure about the file name. Using the original/standard one makes sense. But at the same time, without more context inside `ci/` the file name is not very descriptive. Totally up to you, just a thought, but since we specify the file name when calling `conda mambabuild` maybe naming this `ci/conda-recipe.yml` would be better?

Reply:
Good idea. I renamed it to `ci/conda-recipe.yml`

Reply:
I had to change this back to `meta.yaml` to be recognized by `conda-verify`